vendor/github.com/operator-backing-service-samples/postgresql-operator/pkg/apis/postgresql/v1alpha1/database_types.go

```diff
@@ -28,8 +28,9 @@ type DatabaseStatus struct {
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
 	DBCredentials    string `json:"dbCredentials"`
 	DBConnectionIP   string `json:"dbConnectionIP"`
-	DBConnectionPort int32  `json:"dbConnectionPort"`
+	DBConnectionPort int64  `json:"dbConnectionPort"`
 	DBName           string `json:"dbName"`
+	DBConfigMap      string `json:"dbConfigMap"`
```

Comment:
Why is this change a part of moving PostgreSQL operator PR?

Reply:
It is likely that the operator code cached in the original vendor directory was not up to date... The new content is definitely correct.